Description: Fix for FTBFS with GCC 4.3 bug.
 Fix for Debian FTBFS bug # 461712, where it was failing to build because of
 pre-iso c++ headers that were still being used in the source.
Bug-Debian: http://bugs.debian.org/461712
Author: Peter Karlsson <peterk@debian.org>
Reviewed-By: Robert James Clay <jame@rocasa.us>
Origin: vendor
Last-Update: 2013-06-25

--- multimail-0.49.orig/interfac/main.cc
+++ multimail-0.49/interfac/main.cc
@@ -12,7 +12,7 @@
 #include "interfac.h"
 
 #ifdef USE_NEWHANDLER
-# include <new.h>
+# include <new>
 #endif
 
 #include <locale.h>
@@ -40,7 +40,7 @@ ErrorType::ErrorType()
 	srand((unsigned) starttime);
 
 #ifdef USE_NEWHANDLER
-	set_new_handler(memError);
+	std::set_new_handler(memError);
 #endif
 	origdir = mygetcwd();
 }
